compiler-plugin/src/main/kotlin/arrow/meta/quotes/Quote.kt

```diff
@@ -159,9 +164,10 @@ inline fun <P : KtElement, reified K : KtElement, S> Meta.quote(
         updateFiles(files, fileMutations, match)
         println("END quote.doAnalysis: $files")
         files.forEach {
-          val fileText = it.text
-          if (fileText.contains("//metadebug")) {
-            File(it.virtualFilePath + ".meta").writeText(it.text.replaceFirst("//metadebug", "//meta: ${Date()}"))
+          if (it.text.contains(META_DEBUG_COMMENT)) {
+              File(files.firstOrNull { it.name == DEFAULT_META_FILE_NAME }?.virtualFilePath?.let { path ->
+                  if (it.name != DEFAULT_META_FILE_NAME) path.substring(0, path.lastIndex - 2) + "_${it.name}" else path
+              } + ".meta").writeText(it.text.replaceFirst(META_DEBUG_COMMENT, "//meta: ${Date()}"))
```

Comment:
I think this change wouldn't be necessary because of the comments about the debug command.